### PR TITLE
[SG-998] Move org vault to vault team

### PR DESCRIPTION
**Pull request by differsthecat**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Move org-vault to vault folder and rename extra nested app/vault folder to individual-vault

## Code changes

File moves and import changes only

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/web/src/vault/individual-vault/vault-filter/shared/vault-filter-shared.module.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { NgModule } from "@angular/core";
 
 // TODO refine elsint rule for **/app/shared/*
 // eslint-disable-next-line no-restricted-imports
-import { SharedModule } from "../../../../../app/shared";
+import { SharedModule } from "../../../../app/shared";
```
Comment (MGibson1) [inline apps/web/src/vault/individual-vault/vault-filter/shared/vault-filter-shared.module.ts:5]: Should be able to remove this eslint as well, right?

**Review by MGibson1 — COMMENTED** [review]
couple comments

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,4 +2,10 @@
 #
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
+apps/browser/src/vault @bitwarden/team-vault-dev
+apps/cli/src/vault @bitwarden/team-vault-dev
+apps/desktop/src/vault @bitwarden/team-vault-dev
+apps/web/src/vault @bitwarden/team-vault-dev
 bitwarden_license/bit-web/src/app/secrets-manager @bitwarden/pod-sm-dev
+libs/angular/src/vault @bitwarden/team-vault-dev
+libs/common/src/vault @bitwarden/team-vault-dev
```
[the remark this answers is not in the document]
Comment (differsthecat) [reply .github/CODEOWNERS:11]: I wanted to do this, but there's a comment at the top of the file to sort alphabetically. Probably just keep it alphabetical per section.

File: apps/web/src/auth/settings/emergency-access/emergency-access-attachments.component.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import { AttachmentView } from "@bitwarden/common/vault/models/view/attachment.v
 
 @Component({
   selector: "emergency-access-attachments",
-  templateUrl: "../../../vault/app/vault/attachments.component.html",
+  templateUrl: "../../../vault/individual-vault/attachments.component.html",
```
[the remark this answers is not in the document]
Comment (differsthecat) [reply apps/web/src/auth/settings/emergency-access/emergency-access-attachments.component.ts:16]: Yeah, sorry about that; there was an extra `app` folder that I had that was not necessary at all. Once I removed that, the `vault/vault` folder felt even weirder.

File: apps/web/src/app/core/core.module.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ import { PasswordRepromptService as PasswordRepromptServiceAbstraction } from "@
 
 // TODO refine elsint rule for **/app/core/*
 // eslint-disable-next-line no-restricted-imports
```
[the remark this answers is not in the document]
Comment (differsthecat) [reply apps/web/src/app/core/core.module.ts:28]: Ah yes, good catch!

File: apps/web/src/app/organizations/organization-routing.module.ts [hunk 5]
```diff
@@ -12,13 +12,14 @@ import {
 } from "@bitwarden/common/abstractions/organization/organization.service.abstraction";
 import { Organization } from "@bitwarden/common/models/domain/organization";
 
+import { VaultModule } from "../../vault/org-vault/vault.module";
```
[the remark this answers is not in the document]
Comment (differsthecat) [reply apps/web/src/app/organizations/organization-routing.module.ts:15]: Yeah :/ 
I'm going to do some thinking in to how we can improve this

**Pull request merged**